relnotes/await-timeout.migration.md

```diff
@@ -0,0 +1,4 @@
+## Using `Scheduler` requires linking ebtree
```

Comment:
This is another breaking change in the "not essential" category.

Reply:
This one indeed can be moved to v4.x.x (it is only needed to provide `awaitOrTimeout` via generic scheduler interface) but I took the opportunity to put it here because in practice it is almost not going to matter - pretty much any non-trivial app that uses scheduler also uses its timer utils and thus links ebtree.